src/style.css

```diff
@@ -1,3 +1,63 @@
+:root {
+  --blue-color: #00acdc;
+}
+
 body {
+  box-sizing: border-box;
   margin: 0;
+  font-family: Roboto, Arial, sans-serif;
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. For example, 'Roboto', 'Arial', sans-serif should also include a generic font family like 'sans-serif' as the last fallback.